### DISCUSSION TITLE
Why does "VS" have the name ?, instead of moving away from the dark side?

### DISCUSSION THREAD

**Discussion by sakunix** [VSCodium/vscodium#741] [Ideas]
"VS" is synonymous with Microsoft, I propose to remove the letters "VS" from the name, leave only "Codium" or "IDECodium" or anything that has nothing to do with visual studio and microsoft
regards

**Comment by ghost** [comment]
it's `codium` actually not codim @icomputo

**Comment by daiyam** [comment]
I agree with @icomputo. `VS` should be removed.
With the current issue #746, people are opening issues on `vscode` instead of `vscodium`...
Maybe even if we do the change, some people will still confused but less likely.

**Comment by daiyam** [comment]
answer: #293

**Comment by trymeouteh** [comment]
I am in favor of renaming the app to codium

**Comment by sere3925sere** [comment]
How ambitious for something that merely removes telemetry. Don't you have better things to do instead, like getting more extensions to work?

If you rename it to Codium, it will confuse new users. I'm telling you this as a new user, I will be confused.
